.github/workflows/dotnetcore.yml

```
@@ -0,0 +1,25 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.100
+    - name: Build with dotnet
+      run: dotnet build --configuration Release
+    - name: Test with dotnet test
+      run: dotnet test /p:Configuration=Release --no-build
```

Consistency nitpick:

```suggestion
      run: dotnet test --configuration Release --no-build
```